utils/node_local_test.py

```diff
@@ -1342,6 +1342,46 @@ def test_cache(self):
 
         destroy_container(self.conf, self.pool, container)
 
+    def test_two_mounts(self):
+        """Create two mounts, and check that a file created in one
+        can be read from the other"""
+
+        dfuse0 = DFuse(self.server,
+                       self.conf,
+                       caching=True,
+                       pool=self.pool,
+                       container=self.container)
+        dfuse0.start(v_hint='two_0')
+
+        dfuse1 = DFuse(self.server,
+                       self.conf,
+                       caching=True,
+                       path=os.path.join(self.conf.dfuse_parent_dir,
+                                         'dfuse_mount_1'),
+                       pool=self.pool,
+                       container=self.container)
+        dfuse1.start(v_hint='two_1')
+
+        file0 = os.path.join(dfuse0.dir, 'file')
+        fd = open(file0, 'w')
+        fd.write('test')
+        fd.close()
+
+        file1 = os.path.join(dfuse1.dir, 'file')
+        fd = open(file1, 'r')
+        data = fd.read()
```

Comment:
Should `data` here be the same as was written?
Locally, this is what I see:
```
$ mkdir dfuse1 && dfuse -m dfuse1
$ mkdir dfuse2 && dfuse -m dfuse2
$ echo test > dfuse1/$POOL/$CONT/file && cat dfuse2/$POOL/$CONT/file
test
$ echo updated > dfuse1/$POOL/$CONT/file && cat dfuse2/$POOL/$CONT/file
test
```

Reply:
Yes it should, I just didn't test for that.

Reply:
Done.

Reply:
What about on the case of update? (like with the echo + cat example above)
I'm genuinely asking - not sure if it's expected to be in-sync
For example:
```
        // First write
        file0 = os.path.join(dfuse0.dir, 'file')
        fd = open(file0, 'w')
        fd.write('test')
        fd.close()

        // First read. dfuse1 does not yet have this cached
        file1 = os.path.join(dfuse1.dir, 'file')
        fd = open(file1, 'r')
        data = fd.read()
        fd.close()
        print(data)
        assert data == 'test'

        // Second write to same file
        fd = open(file0, 'w')
        fd.write('updated')
        fd.close()

        // Second read. dfuse1 has this cached from the first read
        fd = open(file1, 'r')
        data = fd.read()
        fd.close()
        print(data)
        assert data == 'updated'
```

Reply:
Makes sense, thanks!